wasmcloud_basics/wash_intro/ctl.md

```diff
@@ -0,0 +1,96 @@
+```
+
+##### Important Fields
+
+In this example, you'll notice in the middle of the output `No actors found`. So far, we do not have any actors running in this environment. Next, we will start the `table_tennis` actor and you will be able to see it here.
+
+Below the actor output is running provider information.
+[TODO: Describe what the 2 items in the provider section are for.]
+
+### wash ctl start actor
+
+In order to start an actor (or provider), we must envoke the `wash ctl start` command. This is the first step to running our business logic in the `wasmcloud` ecosystem.
+
+#### Example
+
+`ctl start actor wasmcloud.azurecr.io/echo:0.2.1`{{execute}}
+
+> Note: Since we have not added our `table_tennis` actor to the registry yet, we will start the `echo` actor that is in the `wasmcloud` container registry.
+
+The output will confirm that we have started an actor. Now, we will be able to check out inventory again and see an actor running.
+
+`ctl get inventory <HOST-ID>`{{copy}}
+
+> Note: You can use the up arrow within the wash REPL to re-run commands from your history.
+
+```bash
+<SNIP>
+
+Actor ID                                                   Image Reference
+MBCFOPM6JW2APJLXJD3Z5O4CN7CPYJ2B4FTKLJUR5YR5MITIU7HD3WD5   wasmcloud.azurecr.io/echo:0.2.1
+
+<SNIP>
+```
+
+### wash ctl start provider
+
+Just like starting an actor, you can start a provider with the `wash ctl start` command. To know what provider we should start for the echo actor, we can inspect it and see what capabilities it has.
+
+> Note: Remember from the previous screen, you can use `wash inspect` to see what capabilities an actor has. The `echo` actor has the HTTP Server capability.
+
+#### Example
+
+`ctl start provider wasmcloud.azurecr.io/httpserver:0.12.1`{{execute}}
+
+Once again, you will see the output that your provider started successfully and you can verify it by checking the inventory.
+
+### wash ctl link
+
+Now that you have the actor and the provider running, you need to tell them to communication with con another. This is done in the linking process via `wash ctl link`. We are going to tell the `wasmcloud` runtime that we want our actor talking with out provider, and give it a few settings to do to.
+
+> Note: In this example, you will need both the ACTOR ID and the PROVIDER ID that is provided from the inventory output. If you run the `get inventory` command, you will have that information in the output window while you are typing the link command.
+
+#### Example
+
+`ctl link <ACTOR-ID> <PROVIDER-ID> wasmcloud:httpserver PORT=8080`{{copy}}
+
+If you were to view the help for this command, you would see that the `wasmcloud:httpserver` piece is refered to as the "contract ID". These contracts can be found in this [repo](https://github.com/wasmCloud/capability-providers).
+
+At this point, if you were to open a new terminal and `curl localhost:8080`, you would see your request echoed back to you.
```

Comment:
At this point, click the "+" icon next to the `Terminal` tab at the top of this page and type `curl localhost:8080` to see your request echoed back to you!